framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -988,46 +1075,57 @@ protected boolean visitTypevarSupertype(
   }
 
   /**
-   * A type variable is a subtype if its upper bounds is below the supertype. Note: When comparing
-   * two type variables this method and visitTypevarSupertype will combine to isValid the subtypes
+   * A type variable is a subtype if its upper bound is below the supertype. Note: When comparing
+   * two type variables, this method and visitTypevarSupertype will combine to isValid the subtypes
    * upper bound against the supertypes lower bound.
    */
   protected boolean visitTypevarSubtype(
```

Comment:
The naming of `visitTypevarSupertype` and `visitTypevarSubtype` feels inconsistent with the rest of the class.  Usually a `visit` method name contains the types of its two arguments, so I expected these to be something like `visitType_TypeVariable` and `visitTypeVariable_Type`.  Can you explain the reason for the naming, or make it more consistent?  (I realize these are not overloads, but more consistency is still desirable if their names start with `visit` rather than, say, `isSubtype`.)

Reply:
https://github.com/typetools/checker-framework/pull/4768